### DISCUSSION TITLE
How can I silence the Ganache server running on the background when I am running my test suite?

### DISCUSSION THREAD

**Discussion by kindaro** [ConsenSys-archive/ganache#4489] [Q&A] [answered]
Consider this code: <https://github.com/kindaro/blockchain-examples/tree/830fe50c7b07af71d1715737147ac713a99130de>. This is the output I see when running `npm run check`: <https://gist.github.com/kindaro/7b8957132c0f6ecc3950e66de4666af6>. I am bothered by lines such as:

```
eth_accounts
eth_gasPrice
eth_blockNumber
eth_sendTransaction
```

As I understand, what happens is that, when required, Ganache spins up a server that, by default, says, for every JSON RPC request made to it, its name.

Is there a way to require Ganache in such a way that these messages be suppressed?

I read that in a _«standalone»_ instance of Ganache, running from command line, these messages can be suppressed with the flag `--quiet`. Is there a way to trigger this flag when requiring Ganache programmatically?

**Comment by davidmurdoch** [comment, marked as answer]
`const provider = Ganache.provider({ logging: { quiet: true }});` should do the trick, unless you are passing your own logger (`{ logging: { logger: console }}`) in which case Ganache does still send the logs to your custom logger.

**Reply by kindaro** [reply]
Thank you. Is this documented somewhere?

**Reply by davidmurdoch** [reply]
Doesn't look like it. Sorry about that!
